src/main/java/com/minecolonies/coremod/util/ServerUtils.java

```diff
@@ -1,147 +1,147 @@
-        return null;
-    }
-}
+package com.minecolonies.coremod.util;
+
+import com.minecolonies.coremod.colony.permissions.Permissions;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.world.World;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Utility for server related stuff.
+ * <p>
+ * Here you can query players on the server.
+ *
+ * @since 0.2
+ */
+public final class ServerUtils
+{
+
+    /**
+     * Private constructor to hide the implicit public one.
+     */
+    private ServerUtils()
+    {
+    }
+
+    /**
+     * Returns the online EntityPlayer with the given UUID.
+     *
+     * @param world world the player is in
+     * @param id    the player's UUID
+     * @return the Player
+     */
+    @Nullable
+    public static EntityPlayer getPlayerFromUUID(@NotNull final World world, @NotNull final UUID id)
+    {
+        for (int i = 0; i < world.playerEntities.size(); ++i)
+        {
+            if (id.equals((world.playerEntities.get(i)).getGameProfile().getId()))
+            {
+                return world.playerEntities.get(i);
+            }
+        }
+        return null;
+    }
+
+    /**
+     * Returns a list of online players whose UUID's match the ones provided.
+     *
+     * @param world the world the players are in.
+     * @param ids   List of UUIDs
+     * @return list of EntityPlayers
+     */
+    @NotNull
+    public static List<EntityPlayer> getPlayersFromUUID(@NotNull final World world, @NotNull final Collection<UUID> ids)
+    {
+        @NotNull final List<EntityPlayer> players = new ArrayList<>();
+
+        for (final Object o : world.playerEntities)
+        {
+            if (o instanceof EntityPlayer)
+            {
+                @NotNull final EntityPlayer player = (EntityPlayer) o;
+                if (ids.contains(player.getGameProfile().getId()))
+                {
+                    players.add(player);
+                    if (players.size() == ids.size())
+                    {
+                        return players;
+                    }
+                }
+            }
+        }
+        return players;
+    }
+
+    /**
+     * Returns a list of players from a list of {@link Permissions.Player}.
+     * <p>
+     * The {@link Permissions.Player} is a wrapper around a {@link UUID} of minecraft players.
+     * The List will simply be converted into an {@link EntityPlayer} type.
+     * <p>
+     * Uses {@link ServerUtils#getPlayerFromPermPlayer(Permissions.Player, World)}.
+     *
+     * @param players The list of players to convert.
+     * @return A list of {@link EntityPlayer}s
+     */
+    @NotNull
+    public static List<EntityPlayer> getPlayersFromPermPlayer(@NotNull final List<Permissions.Player> players, @NotNull final World world)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): world [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)